docs/docusaurus/docs/cloud/connect/connect_postgresql.md

```diff
@@ -0,0 +1,179 @@
+---
+sidebar_label: 'Connect GX Cloud to PostgreSQL'
+title: 'Connect GX Cloud to PostgreSQL'
+description: Connect GX Cloud to a PostgreSQL Data Source.
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To validate data stored in a PostgreSQL data warehouse from GX Cloud, you must add the GX Agent to your deployment environment. The GX Agent acts as an intermediary between GX Cloud and PostgreSQL and allows you to securely access and validate your data in GX Cloud.
+
+New to GX Cloud and not sure that it's the right solution for your organization? See [Try GX Cloud](../try_gx_cloud.md).
+
+## Prerequisites
+
+- You have a [GX Cloud Beta account](https://greatexpectations.io/cloud) with [Admin or Editor permissions](../about_gx.md#roles-and-responsibilities).
+
+- You have a PostgreSQL database, schema, and table.
+
+- You have a [PostgreSQL instance](https://www.postgresql.org/download/). To improve data security, GX recommends using a separate user service account to connect to GX Cloud.
+
+- You know your PostgreSQL access credentials.
+
+- You have stopped all local running instances of the GX Agent.
```

Comment:
Why is this a pre-req?
It shouldn't matter if an agent is running or not unless you need to set new postgres specific data source ENV vars.

Reply:
@Kilo59 this was added after a discussion with @allensallinger. Asking users to stop all running instances of the GX Agent reduces the likelihood of issues.